### PR TITLE
Restore ASMFS library to working order

### PR DESCRIPTION
**Pull request by juj**

### CODE REVIEW

File: tools/system_libs.py [hunk 1]
```diff
@@ -1137,7 +1137,7 @@ def get_files(self):
   def can_build(self):
     # ASMFS is looking for a maintainer
```
Comment (kripken) [inline tools/system_libs.py]: Do you intend to work on ASMFS? If so let's remove this comment. Perhaps also enable tests?

Comment (juj) [reply tools/system_libs.py]: Yes, at some point. Not yet able to give it full attention, but this was enough to make my 3D engine work again on latest Emscripten master.

Checking out our asmfs tests in browser suite, unfortunately they do not yet work with this.

**Comment by juj** [comment]
There are tests, but they are currently disabled, needs more fixing.

File: tools/system_libs.py (hunk 1 above)
Comment (kripken) [reply tools/system_libs.py]: I see, thanks. lgtm

**Pull request merged**